[5.11.0] - Remove unnecessary references and promote stored procedures (#2891)
wso2/docs-is: en/docs/setup/data-purging.md

    For more information about session persistence, see [Authentication Session Persistence](../../learn/authentication-session-persistence).

!!! tip

>     It is recommended to run these steps at the time when server traffic is low. Especially if you are running this in the production environment for the first time, since the data volume to be purged may be higher. However, consider this as a housekeeping task that needs to be run at regular intervals. 

Reviewer: ```suggestion
    It is recommended to run these steps at a time when server traffic is low. This recommendation is particularly important if you are running these steps in the production environment for the first time because the data volume to be purged may be higher. However, consider this as a housekeeping task that needs to be run at regular intervals. 
```

    You can schedule the aforementioned queries to be run at regular intervals.
    

> 1. Disable the internal session cleanup process by configuring the following properties in the `deployment.toml` file found in the `<IS_HOME>/repository/conf` folder.

Reviewer: ```suggestion
1. Disable the internal session cleanup process by configuring the following properties in the `deployment.toml` file (which is stored in the `<IS_HOME>/repository/conf` folder).
```

   , and
    [Oracle](https://docs.oracle.com/cd/E11882_01/backup.112/e10642/rcmbckba.htm#BRADV8138)
    offical documentation.
1. Disable the internal session cleanup process by configuring the following properties in the `deployment.toml` file (which is stored in the `<IS_HOME>/repository/conf` folder).

    !!! tip
        We recommend that you test the database dump before the cleanup task as the cleanup can take some time.
    ```toml
    [session_data.cleanup]
    enable_expired_data_cleanup = false
    clean_logged_out_sessions_at_immediate_cycle = false
    enable_pre_session_data_cleanup = false
    
    ```

3.  To clean the session and operation data that are stored in the `IDN_AUTH_SESSION_STORE` table, run the following script on the database dump. 
2. To clean the session and operation data that are stored in the `IDN_AUTH_SESSION_STORE` table, run the required database script on the database based on the database type. 

    !!! note
        This script clears session data that are older than the last 14 days and operational data that are older than the last 6 hours.

>     ??? tip "Click to view the MySQL script" 

Reviewer: Is it necessary to remove this? This seems to be an example for MySQL.
@deshankoswatte

Reviewer: Okay. I think I found the answer. Since we have pointed to all the DB scripts in the below note, I suppose this is redundant. +1
